Nagstamon/Objects.py

```diff
@@ -147,6 +147,7 @@ class GenericService(GenericObject):
 
     def __init__(self):
         GenericObject.__init__(self)
+        self.unreachable = False
```

Comment:
Sorry for the late reply.

Why has self.unreachable to be initialized here? Shouldn't it be set like other properties?

Reply:
Because it applies to services only. We can move it up if you want